<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationMain</class>
 <widget class="QWidget" name="CalibrationMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>561</width>
    <height>528</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SMBX64 Playable Sprite Calibrator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="_resourses/images.qrc">
    <normaloff>:/images/Icon16.png</normaloff>:/images/Icon16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="22" column="0">
    <widget class="QPushButton" name="AnimatorButton">
     <property name="toolTip">
      <string>Edit animations set</string>
     </property>
     <property name="text">
      <string>Animator</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QPushButton" name="CopyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Copy this frame into internal clipboard</string>
     </property>
     <property name="text">
      <string>copy</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="FrameY">
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Offset Y</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="editSizes">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="22" column="3" colspan="2">
    <widget class="QPushButton" name="MakeTemplateB">
     <property name="toolTip">
      <string>Useful for tests or for cration from scratch</string>
     </property>
     <property name="text">
      <string>Generate &quot;Square Man&quot; template</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Physics calibration:</string>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <widget class="QPushButton" name="PasteButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Paste from internal clipboard</string>
     </property>
     <property name="text">
      <string>Paste</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="2">
    <widget class="QSpinBox" name="OffsetY">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Frame X</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current frame (10x10):</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QPushButton" name="AboutButton">
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="2">
    <widget class="QCheckBox" name="isDuckFrame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Is duck frame</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Height (duck)</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QSpinBox" name="Height_duck">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="28" column="3" colspan="2">
    <widget class="QPushButton" name="calibrateImage">
     <property name="toolTip">
      <string>Calibrating frames relative to exists physics (useful for SMBX where you can't edit physics)</string>
     </property>
     <property name="text">
      <string>Image frames calibration</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QSpinBox" name="grabOffsetY">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Grab offset X</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QSpinBox" name="grabOffsetX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Grab offset Y</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Global:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="2">
    <widget class="QSpinBox" name="OffsetX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="21" column="3">
    <widget class="QPushButton" name="SaveConfigButton">
     <property name="text">
      <string>Save config</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Offset X</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QPushButton" name="applyToAll">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Apply to all</string>
     </property>
    </widget>
   </item>
   <item row="21" column="4">
    <widget class="QPushButton" name="OpenSprite">
     <property name="text">
      <string>Open sprite</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Frame Y</string>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <widget class="QPushButton" name="Matrix">
     <property name="toolTip">
      <string>Select frame from full sprite matrix</string>
     </property>
     <property name="text">
      <string>Matrix</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QCheckBox" name="EnableFrame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>237</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Enable this frame</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="FrameX">
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QSpinBox" name="Width">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QSpinBox" name="Height">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="3">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>GFX offsets of frame:</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1" colspan="2">
    <widget class="QCheckBox" name="isRightDirect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Right direction</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1" colspan="2">
    <widget class="QCheckBox" name="showGrabItem">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Show carried item</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="20" colspan="2">
    <widget class="QGraphicsView" name="PreviewGraph"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="_resourses/images.qrc"/>
 </resources>
 <connections/>
</ui>
